tests/wpt/metadata/html/browsers/windows/auxiliary-browsing-contexts/opener-noreferrer.html.ini

```diff
@@ -1,5 +1,6 @@
 [opener-noreferrer.html]
   type: testharness
+  expected: TIMEOUT
```

Comment:
TIMEOUT because we're navigating the main browsing context(while the test calls for navigating '_blank'),   as anchor elements currently ignore their 'target'(it's actually addressed in https://github.com/servo/servo/pull/20678). 

Since this PR properly adds unloading/discarding the document, the test now timeouts because the event handler on the main page is discarded along with the whole page(we should be navigating a new auxiliary browsing context instead).

Reply:
It looks like this PR is causing the first document to be discarded, which shouldn't happen since, like you said, the first document should be salvageable.

Reply:
Yep, I'm investigating...